frontend/app/page.tsx

```diff
@@ -123,10 +124,23 @@ export default function Home() {
                 />
               </label>
 
-              <label className="label-row">
+              <label>
                 Text
                 <input type="text" />
               </label>
+
+              <label>
+                Textarea
+                <textarea rows={5} />
+              </label>
+
+              <button>Yellow</button>
+              <button className="button-pink">Pink</button>
+              <button className="button-cyan">Cyan</button>
```

Comment:
focus menee toisten buttonien ja textarean päälle. Pitäiskö buttonilla olla jonkun verran marginia?

Reply:
Voisi joo laittaa varalta. Meillä ei pitäisi olla suunnitelmien mukaan missään buttoneita niin lähekkäin mitä tuossa etusivun esimerkkibuttoneissa ja -inputeissa, mutta marginilla ainakin saa pidettyä huolen ettei sellaista tilanentta tulekaan. Korjailen!